config/hosts_test.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestConfig_GetKDCsUsesConfiguredKDC(t *testing.T) {
+	t.Parallel()
+
+	// This test is meant to cover the fix for
+	// https://github.com/jcmturner/gokrb5/issues/332
+	krb5ConfWithKDCAndDNSLookupKDC := `
+[libdefaults]
+ dns_lookup_kdc = true
+
+[realms]
+ TEST.GOKRB5 = {
+  kdc = 10.1.2.3.4:88
```

Comment:
This is not a valid IP address (5 octets). I guess it does not matter for this test as it is just being treated as a string but I think it would be better to use a valid one to save future confusion.